server/src/main/java/org/elasticsearch/common/unit/RelativeByteSizeValue.java

```diff
@@ -90,7 +90,7 @@ public static RelativeByteSizeValue parseRelativeByteSizeValue(String value, Str
 
     public String getStringRep() {
         if (ratio != null) {
-            return ratio.toString();
+            return RatioValue.formatNoTrailingZerosPercent(ratio.getAsPercent());
```

Comment:
Seems weird that this isn't an instance method on `RatioValue`.

Reply:
I see I can convert it to an instance method. I will do that. I am cc'ing @gmarouli (who implemented this) in case there is an objection.

Reply:
When we first created this it was processing doubles only, later as we adopted the `RelativeByteSizeValue` that wasn't necessary but we chose to preserve as static because we thought it would be useful for others too. I do not have strong objections.

Reply:
Oh thanks for the explanation! For now, I made it an instance method. But it can be converted back to a static method when somebody needs it. Please feel free to tell me if this is not sufficient and unresolve this conversation.

Reply:
Great! No, it's sufficient, I just saw the comment and I wanted to give some context :)